File: ejercicios-js/ejercicio3.js
const palabras = ["Aceituna", "Murciélago", "Educación", "Aeropuerto", "Otorrinolaringólogo", "Euforia", "Aceite", "Paleontólogo", "Arquitectura", "Hipopótamo"];
const vocales = ['a', 'e', 'i', 'o', 'u', 'á', 'é', 'í', 'ó', 'ú'];



function getRandomInt(min, max) {
  return Math.floor(Math.random() * (max - min + 1)) + min;
}


const palabra = palabras[getRandomInt(0, palabras.length - 1)];

let contador = 0;
for (const letra of palabra.toLowerCase()) {
  if (vocales.includes(letra)) {
    contador++;
  }
}

console.log(`La palabra seleccionada es "${palabra}" y tiene ${contador} vocales.`);

File: ejercicios-py/ejercicio3.py
import random


palabras = [
    "Aceituna",
    "Murciélago",
    "Educación",
    "Aeropuerto",
    "Otorrinolaringólogo",
    "Euforia",
    "Aceite",
    "Paleontólogo",
    "Hipopótamo",
    "Arquitectura"
]


def contar_vocales(palabra):
    vocales = "aeiouáéíóúAEIOUÁÉÍÓÚ"
    lista_vocales = [letra for letra in palabra if letra in vocales]
    return lista_vocales


palabra_seleccionada = random.choice(palabras)
vocales_encontradas = contar_vocales(palabra_seleccionada)

print(f"Palabra seleccionada: {palabra_seleccionada}")
print(f"Vocales encontradas: {', '.join(vocales_encontradas)}")
print(f"Cantidad de vocales: {len(vocales_encontradas)}")
